srcpkgs/nwipe/template

```diff
@@ -1,7 +1,9 @@
 # Template file for 'nwipe'
 pkgname=nwipe
-version=0.25
+version=0.26
 revision=1
+# nwipe.c:29:10: fatal error: bits/pthreadtypes.h: No such file or directory
+archs="~*-musl"
```

Comment:
Please use `broken` for this.
Nevertheless, this should(?) be fixable by changing
```diff
-#include <bits/pthreadtypes.h>
-#include <bits/sigthread.h>
+#include <sys/types.h>
```
in `src/nwipe.c`.
(Not sure about this, just skimmed through the headers.)

Reply:
Thank you!